client/src/pages/SignUp/functions.jsx

```diff
@@ -16,12 +16,36 @@ export function validatePasswordLength(password) {
   return true;
 }
 
-export async function signUpUser(user) {
-  console.log(signUpUser);
-  /*eslint no-undef: 0*/
-  let promise = new Promise((res, rej) => {
-    setTimeout(() => res('An error occured!'), 1000);
+import axios from "axios";
+
+
+export function signUpUser(user) {
+
+  const result = axios({
+    method: 'post',
+    url: 'http://localhost:5001/user/signup',
```

Comment:
Please don't use hardcoded urls for any requests - these will only work for your local machine!